RecoTracker/IterativeTracking/python/ElectronSeeds_cff.py

```diff
@@ -246,7 +246,12 @@
     'pixelLessStepSeeds',
     'tripletElectronSeeds',
     'pixelPairElectronSeeds',
-    'stripPairElectronSeeds'
+    'stripPairElectronSeeds',
+    'lowPtTripletStepSeeds',
+    'lowPtQuadStepSeeds',
```

Comment:
so as Matti suggested and I then proved in the reply to him, the low pt seeds significantly help. The detached seeds dont help that much but I would prefer to have them unless timing forces me not to.